src/styles/accountConnection.styl

```diff
@@ -20,6 +21,21 @@
         flex-direction column
         justify-content center
 
+        .col-konnector-update
+            width 90%
+            align-self center
+            border 1px solid yourPink
+            background-color chablis
+            color monza
+            border-radius 5px
+            padding .5rem
+            cursor pointer
+
+            &:hover,
+            &:active,
+            &:focus
+                text-decoration underline
+
```

Comment:
This should a generic `.alert` class, see how bootstrap handles this : https://getbootstrap.com/docs/4.2/components/alerts/#examples